frontend/src/components/Table/MoneyCell.tsx

```diff
@@ -6,11 +6,12 @@ import { CellProps } from 'react-table';
 import { formatMoney } from 'utils';
 import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
 import TooltipWrapper from 'components/common/TooltipWrapper';
+import { Form } from 'react-bootstrap';
 
 interface IEditableCellProps {
   namespace: string;
   suppressValidation: boolean;
-  cell: CellProps<IProperty, number>;
+  cell: CellProps<IProperty, number | ''>;
 }
 
 export const EditableMoneyCell = ({
```

Comment:
I may need to apply my addition to stop allowing editing the land assessed value for buildings here.